Snippets/ABSDataBus/ABSDataBus_1/app.config

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
```

Comment:
Effing nuget keeps adding that shit. I'll remove them when I update to the beta (about to push it to myget)